Sources/HTTP/HTTPServer.swift

```diff
@@ -1,54 +1,336 @@
-// This source file is part of the Swift.org Server APIs open source project
 //
-// Copyright (c) 2017 Swift Server API project authors
-// Licensed under Apache License v2.0 with Runtime Library Exception
+//  HTTPServer.swift
+//  HTTP
 //
-// See http://swift.org/LICENSE.txt for license information
+//  Created by Helge Hess on 22.10.17.
```

Comment:
We probably want to keep the swift.org license stuff, right?

Reply:
Nah, I wrote it. It should have my name on it 😬 License is fine though.

Reply:
This is probably not a good idea.
None of the other files in this repo have neither this nor "Copyright © 2017 SomeCompany..."
Do you think each time someone changes this file, they should add "Changed by..." to it as well? We have a git history for it, so your work won't be forgotten :)